.github/workflows/ci.yml

```diff
@@ -65,27 +77,27 @@ jobs:
 
       - name: Check formatting
         if: matrix.scala != '3.1.0'
-        run: sbt ++${{ matrix.scala }} '${{ matrix.ci }}' scalafmtCheckAll
```

Comment:
This `${{ matrix.ci }}` is what was used to to set the root project to either `rootJVM` or `rootJS`.

Reply:
I doubt Ross would mind if you felt like pushing a fix 😄

Reply:
Oh, something in the merge made it look like that had been intentionally removed, and I didn't understand it, so I played along.